test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -12860,6 +12860,24 @@ def add_config(self, conf={}, hup=True):
 
         return self.apply_config(conf, hup)
 
+    def unset_mom_config(self, name, hup=True):
```

Comment:
I would suggest to rename this as delete_config (as we have add_config) or rename add_config to set_mom_config